client/components/section-nav/tabs.jsx

```diff
@@ -42,18 +43,18 @@ class NavTabs extends Component {
 	};
 
 	componentDidMount() {
-		this.setDropdown();
-		this.debouncedAfterResize = debounce( this.setDropdown, 300 );
-
-		window.addEventListener( 'resize', this.debouncedAfterResize );
+		this.setDropdownAfterLayoutFlush();
+		window.addEventListener( 'resize', this.setDropdownDebounced );
 	}
 
 	componentDidUpdate() {
-		this.setDropdown();
+		this.setDropdownAfterLayoutFlush();
 	}
 
 	componentWillUnmount() {
-		window.removeEventListener( 'resize', this.debouncedAfterResize );
+		window.removeEventListener( 'resize', this.setDropdownDebounced );
+		this.setDropdownDebounced.cancel();
```

Comment:
Should we also add a link back to documentation, or do you think folks will understand? https://lodash.com/docs/4.17.4#debounce